go/vt/wrangler/vdiff.go

```diff
@@ -466,6 +478,64 @@ func findPKs(table *tabletmanagerdatapb.TableDefinition, targetSelect *sqlparser
 	return orderby, nil
 }
 
+// getColumnCollations determines the proper collation to use for each
+// column in the table definition leveraging MySQL's collation inheritence
+// rules.
+func getColumnCollations(table *tabletmanagerdatapb.TableDefinition) (map[string]collations.Collation, error) {
+	collationEnv := collations.Local()
+	createstmt, err := sqlparser.Parse(table.Schema)
+	if err != nil {
+		return nil, err
+	}
+	createtable, ok := createstmt.(*sqlparser.CreateTable)
```

Comment:
I don't know, so I'm asking/verifying: is there any chance for a `CreateView` at this point?

Reply:
Hmmm, not today. But in the future there could be.